src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/service/BlobSplitterImpl.java

```diff
@@ -90,15 +88,19 @@ private boolean splitRtdTaeBlob(BlobApplicationAware blob, String blobPath,
 
     int chunkNum = 0;
     String chunkName;
+    String checkSum = "";
     MutableBoolean isChecksumSkipped = new MutableBoolean(checksumSkipped);
 
     try (
         LineIterator it = FileUtils.lineIterator(
             Path.of(blobPath).toFile(), "UTF-8")) {
       if (it.hasNext() && isChecksumSkipped.isFalse()) {
-        String line = it.nextLine();
-        log.info("Checksum: {} {}", blob.getBlob(), line);
-        blob.getReportMetaData().setCheckSum(line);
+        checkSum = it.nextLine();
+        if (!checkSum.matches("^#sha256.*")) {
```

Comment:
what about extract the regex into a const?
if the checksum does not adhere the convention, we log an error message and go on? the file processing is not blocked or interrupted, right?

Reply:
Yep regex in a constant will be better.🙏
For now, we just want to know if the checksum had an error, but we can continue the process because from what I know we don't have strict constraints for this field, if I'm wrong we can break it here.👍